pnpm-lock.yaml

```diff
@@ -149,10 +149,10 @@ importers:
       jest-fetch-mock: 3.0.3
```

Comment:
I wouldn't expect this PR to have any changes from the `master` branch. Can you revert this file? (or just check in the one from master)